### ISSUE TITLE
NoSuchMethodError: `java.nio.FloatBuffer.clear()Ljava/nio/FloatBuffer`

### ISSUE THREAD

**Issue by Boundarybreaker** [Ladysnake/Satin#8]
```java
java.lang.NoSuchMethodError: java.nio.FloatBuffer.clear()Ljava/nio/FloatBuffer;
	at ladysnake.satin.api.util.GlMatrices.getTmpBuffer(GlMatrices.java:48) ~[Satin-1.4.0@net.fabricmc.yarn.1_16_1.1.16.1+build.1-v2.jar:?]
	at ladysnake.satin.api.util.GlMatrices.getInverseTransformMatrix(GlMatrices.java:111) ~[Satin-1.4.0@net.fabricmc.yarn.1_16_1.1.16.1+build.1-v2.jar:?]
```

`FloatBuffer#clear` is synthetic, with the parent returning `Buffer`. For some reason, the bytecode thinks it should be returning a FloatBuffer, even though the result is ignored.

**Comment by Pyrofab** [comment]
Should be fixed with latest release.

**Comment by Pyrofab** [comment]
Should *actually* be fixed since dee37ea4fe2fc12aac360e4802995e1541547f0a
